scene/3d/spatial.cpp

```diff
@@ -676,7 +676,7 @@ void Spatial::set_identity() {
 void Spatial::look_at(const Vector3 &p_target, const Vector3 &p_up) {
 
 	Transform lookat;
-	lookat.origin = get_global_transform().origin;
+	lookat = get_global_transform();
```

Comment:
You can move that up to the previous line as `Transform lookat = get_global_transform();`.

Reply:
Done.